src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -87,6 +92,7 @@ public function __construct(array $defaultOptions = [], int $maxHostConnections
      */
     public function request(string $method, string $url, array $options = []): ResponseInterface
     {
+        $this->logger->debug('Making a request', ['url' => $url, 'method' => $method, 'client' => static::class]);
```

Comment:
that one may be at a higher level than the debug message telling you which client implementation was used.